lib/cf-socket.c

```diff
@@ -79,7 +79,7 @@
 #include "memdebug.h"
 
 
-#define DEBUG_CF 0
+#define DEBUG_CF 1
```

Comment:
Did you mean to enable this by default?

Reply:
Leftover from my debugging. Should change that.